src/screens/Topics/index.js

```diff
@@ -176,7 +176,7 @@ const Topics = () => {
             <Button
               onPress={() => next()}
               disabled={!(topicSelected.length >= minTopic)}
-              style={topicSelected.length >= minTopic ? null : styles.button}>
+              styles={topicSelected.length >= minTopic ? null : styles.button}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `styles` prop has been renamed from `style`. This could potentially cause issues if the `Button` component does not have a `styles` prop. Please ensure that the `Button` component accepts a `styles` prop, otherwise this change will not have any effect.
```diff
- styles={topicSelected.length >= minTopic ? null : styles.button}>
+ style={topicSelected.length >= minTopic ? null : styles.button}>
```
Also, it's worth noting that passing `null` to the `style` prop when `topicSelected.length >= minTopic` is true might not be the best approach. If there are default styles that should be applied in this case, consider defining them and passing them instead of `null`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->